### ISSUE TITLE
[Android] Video music playing when in background

### ISSUE THREAD

**Issue by Joshy-2010** [mmatyas/pegasus-frontend#1018]
### The issue

Nothing to bad: Its just a bit annoying that video (preview video for games) music keeps playing then it really shouldn't: for example, if I open the pegasus settings and happen to have a game which has a video selected, it still keeps playing the video. This also applies if I open an already running app from android "running apps" list (this 3rd navigation button next to home and back) instead of through pegasus, and happen to have a game which has a video selected.

TLDR: If you have a game which has a video selected and put pegasus in the background, the video continuous playing (with annoying audio)

### System info

Android 10 on an Axon 10 Pro, pegasus selected as default launcher

### Pegasus version

Latest stable version downloaded from the website 5 mins agoalpha16-42-g996720eb (2022-08-02)
